client/src/__tests__/components/grants/List.test.tsx

```diff
@@ -213,15 +212,28 @@ describe("<List />", () => {
         });
 
         test("should be visible if user didn't apply yet", async () => {
+          const round = buildRound({ address: "0x1234" });
+
+          store.dispatch(web3ChainIDLoaded(5));
+
+          store.dispatch({
+            type: "ROUNDS_ROUND_LOADED",
+            address: "0x1234",
+            round,
+          });
+
+          store.dispatch({
+            type: "GRANT_METADATA_FETCHED",
+            data: buildProjectMetadata({}),
+          });
+
+          store.dispatch({ type: "PROJECTS_LOADED", projects });
+
           store.dispatch({ type: "ROUND_APPLICATION_NOT_FOUND", roundAddress });
 
           renderWrapped(<List />, store);
 
-          expect(
-            screen.getByText(
-              "Apply to Optimism Grant Round and get your project funded!"
-            )
-          ).toBeInTheDocument();
+          expect(screen.getByText("Apply to Round")).toBeInTheDocument();
```

Comment:
shall we check `Apply to {round.programName}` so we know we are using the program name?